lib/plugins/wallet/lnd/lnd.js

```diff
@@ -20,7 +21,7 @@ module.exports = {
 }
 
 function connect () {
-  return lnd.connect(options.lnd || {})
+  return lnd.connect(LND || {})
```

Comment:
Is this right? `lnd.connect()` seems to expect an object, but values of environment variables are strings.